libs/tunables/tunables.c

```diff
@@ -0,0 +1,68 @@
+
+#include <avr/eeprom.h>
+#include <avr/io.h>
+
+enum function_type {
+    GETTER = 0,
+    SETTER = 1,
+    MEASURE = 2
+}
+
+// A global variable that gives each board maximum of 16 tunable parameters
+uint32_t TUNABLES_MEM[16] __attribute__((section(".eeprom")))
+= { 0 };
+
+// Can't I just declare send_can in tunables_init??????????????????????????
+void timer0_callback(void) {
+    send_can = true;
+}
```

Comment:
You don't need this. It will be declared in the application file. this is a library file.